commands/call.js

```diff
@@ -17,8 +17,13 @@ module.exports = {
             type: 'string',
             default: '0'
         })
+        .option('base64',  {
+            desc: 'Treat arguments as base64-encoded BLOB.',
```

Comment:
I'd rather not add a new argument, if we can avoid it. Why not inspect `args` to figure out what it is?

* if absent, empty string, or starts with `{`, then JSON
* else, try to decode as base64
  * if this causes an error, throw an error explaining the parsing rules

Reply:
@chadoh I am not sure about this. Maybe we actually want to pass UTF-8 string verbatim when it's not JSON.

Reply:
What do you mean, "pass it verbatim"?

Reply:
@chadoh I mean that nobody prevents you from implementing your contract in a way where expected arg is just string (vs JSON). Like you pass string `foobar` directly as args instead of passing `{"argumentName":"foobar"}`